core/src/main/java/org/apache/iceberg/jdbc/JdbcCatalog.java

```diff
@@ -503,6 +541,78 @@ public boolean namespaceExists(Namespace namespace) {
     return JdbcUtil.namespaceExists(catalogName, connections, namespace);
   }
 
+  @Override
+  public boolean dropView(TableIdentifier identifier) {
+    int deletedRecords =
+        execute(
+            JdbcUtil.dropViewSql(),
+            catalogName,
+            JdbcUtil.namespaceToString(identifier.namespace()),
+            identifier.name());
+
+    if (deletedRecords == 0) {
+      LOG.info("Skipping drop, view does not exist: {}", identifier);
+      return false;
+    }
+
+    LOG.info("Dropped view: {}", identifier);
+    return true;
+  }
+
+  @Override
+  public List<TableIdentifier> listViews(Namespace namespace) {
+    if (!namespaceExists(namespace)) {
+      throw new NoSuchNamespaceException("Namespace does not exist: %s", namespace);
+    }
+    return fetch(
+        row ->
+            JdbcUtil.stringToTableIdentifier(
+                row.getString(JdbcUtil.VIEW_NAMESPACE), row.getString(JdbcUtil.VIEW_NAME)),
+        JdbcUtil.listViewsSql(),
+        catalogName,
+        JdbcUtil.namespaceToString(namespace));
+  }
+
+  @Override
+  public void renameView(TableIdentifier from, TableIdentifier to) {
+    if (!viewExists(from)) {
+      throw new NoSuchViewException("View does not exist");
+    }
+    if (!namespaceExists(to.namespace())) {
```

Comment:
nit: the code convention (which unfortunately isn't enforced by tools) in Iceberg is to add a space after a closing }